src/main/java/Ui.java

```diff
@@ -0,0 +1,9 @@
+public class Ui {
```

Comment:
It would be better if there can be more functionality to represent interaction with users, including the different response messages.